exercises/change/metadata.yml

```diff
@@ -1,2 +1,4 @@
 ---
 blurb: "Correctly determine change to be given using the least number of coins"
+source: "Software Craftsmanship - Kata-logue"
```

Comment:
Maybe snip off "Kata-logue"?  I don't see those words mentioned anywhere.

Reply:
I've changed them to the title on the page